src/net/inv.rs

```diff
@@ -55,6 +55,7 @@ use chainstate::burn::BlockHeaderHash;
 use chainstate::burn::BlockSnapshot;
 
 use chainstate::stacks::db::StacksChainState;
+use chainstate::stacks::StacksBlockHeader;
```

Comment:
Please revert all changes to `src/net/inv.rs`.

Reply:
Sure. Do you mind if I let the `#[derive(Debug)]` though? I did use it for logging and inspecting the PeerNetwork, that was useful and informative.

Reply:
Keeping `#[derive(Debug)]` is fine